2014/01/19 00:33:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "logout".' in /var/www/www.yii-test.local/framework/web/CController.php:483
Stack trace:
#0 /var/www/www.yii-test.local/framework/web/CController.php(270): CController->missingAction('logout')
#1 /var/www/www.yii-test.local/framework/web/CWebApplication.php(282): CController->run('logout')
#2 /var/www/www.yii-test.local/framework/web/CWebApplication.php(141): CWebApplication->runController('auth/logout')
#3 /var/www/www.yii-test.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/www.yii-test.local/messages/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/messages/index.php?r=auth/logout
HTTP_REFERER=http://www.yii-test.local/messages/index.php?r=user/profile
---
2014/01/19 00:35:30 [error] [php] Undefined index: name (/var/www/www.yii-test.local/messages/protected/controllers/AuthController.php:23)
Stack trace:
#0 /var/www/www.yii-test.local/framework/web/CController.php(286): AuthController->runAction()
#1 /var/www/www.yii-test.local/framework/web/CController.php(265): AuthController->runActionWithFilters()
#2 /var/www/www.yii-test.local/framework/web/CWebApplication.php(282): AuthController->run()
#3 /var/www/www.yii-test.local/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/www.yii-test.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#5 /var/www/www.yii-test.local/messages/index.php(13): CWebApplication->run()
REQUEST_URI=/messages/index.php?r=auth/logout
in /var/www/www.yii-test.local/messages/protected/controllers/AuthController.php (23)
in /var/www/www.yii-test.local/messages/index.php (13)
2014/01/19 19:30:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "yii".' in /var/www/www.yii-test.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/www.yii-test.local/framework/web/CWebApplication.php(141): CWebApplication->runController('yii')
#1 /var/www/www.yii-test.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/www.yii-test.local/messages/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/messages/index.php?r=yii
---
2014/01/19 19:49:14 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/www.yii-test.local/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/www.yii-test.local/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/www.yii-test.local/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/www.yii-test.local/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/www.yii-test.local/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/www.yii-test.local/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/www.yii-test.local/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/www.yii-test.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/www.yii-test.local/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/www.yii-test.local/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/users/adm...')
#9 /var/www/www.yii-test.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/www.yii-test.local/messages/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/messages/index.php?r=admin/users/admin
HTTP_REFERER=http://www.yii-test.local/messages/index.php?r=admin/users/index
---
